### PR TITLE
fix: ITO and red packet day-to-day

### PR DESCRIPTION
**Pull request by hanszhoou**
[MASK-467](https://mask.atlassian.net/browse/MASK-467)
[MASK-519](https://mask.atlassian.net/browse/MASK-519)
[MASK-525](https://mask.atlassian.net/browse/MASK-525)
[MASK-535](https://mask.atlassian.net/browse/MASK-535)
[MASK-543](https://mask.atlassian.net/browse/MASK-543)

## Description

Closes # (NO_ISSUE)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Comment by zhouhanseng** [comment]
@DimensionDev/qa

**Comment by wenluomask** [comment]
Please check the issue in https://mask.atlassian.net/browse/MASK-519.

File: packages/mask/src/plugins/ITO/SNSAdaptor/ITO.tsx [hunk 1]
```diff
@@ -405,8 +405,7 @@ export function ITO(props: ITO_Props) {
         (ev) => {
             if (ev.open) return
             if (destructState.type !== TransactionStateType.CONFIRMED) return
-            resetDestructCallback()
-            retryITOCard()
+            window.location.reload()
```
Comment (Jack-Works) [inline packages/mask/src/plugins/ITO/SNSAdaptor/ITO.tsx:408]: Hmm you sure?

Comment (zhouhanseng) [reply packages/mask/src/plugins/ITO/SNSAdaptor/ITO.tsx:408]: I'm sure..

Comment (guanbinrui) [reply packages/mask/src/plugins/ITO/SNSAdaptor/ITO.tsx:408]: @Jack-Works A plugin level reload?

**Comment by wenluomask** [comment]
Checked.

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**